command/eval_list.go

```diff
@@ -0,0 +1,182 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/hashicorp/nomad/api/contexts"
+	"github.com/posener/complete"
+)
+
+type EvalListCommand struct {
+	Meta
+}
+
+func (c *EvalListCommand) Help() string {
+	helpText := `
+Usage: nomad eval list [options]
+
+  List is used to list the set of evaluations processed by Nomad.
+
+General Options:
+
+  ` + generalOptionsUsage(usageOptsDefault) + `
+
+Eval Status Options:
+
+  -verbose
+    Show full information.
+
+  -per_page
+    How many results to show per page.
+
+  -page_token
+    Where to start pagination.
```

Comment:
These should probably be `-per-page` and `-page-token`. I think this is the standard for other double worded flags, like `-force-color`.